index.html

```diff
@@ -22,91 +23,116 @@
-        <p>Each buildpack examines layer metadata from the previous build, re-generates any layer directories that need to be replaced, and updates metadata (if necessary). Buildpacks may choose to modify the app source directory. A transparent, inter-build cache may be used to improve performance and provide other buildpacks with build-time dependencies.</p>
-        <img src="images/build.png" alt="Build Process" class="img-center">
+        <p>Build your application by passing in your application source code directory and image name to the <code class="highlight">pack build</code> command.</p>
 
-        <h3 class="mt-5 pt-4">Export</h3>
-        <p>The remote app layer and any re-generated buildpack layers are replaced. The config layer is updated with new metadata. This process uses image layer rebasing, which can be accomplished remotely against a Docker v2 registry.</p>
-        <img src="images/export.png" alt="Export Process" class="img-center">
+        <pre class="command-line code-block"><code>pack build --path &lt;path to source code&gt; &lt;repository name&gt;:&lt;image tag (optional)&gt;</code></pre>
 
-        <h3 class="mt-5 pt-4">Security Patching</h3>
-        <p>When CVEs hit, certain image layers can be patched without changing app behavior. These layers can be rebased en-mass using remote image layer rebasing on recent Docker registries. This is a metadata change that requires minimal data transfer.</p>
+        <p>Upon a successful build, you will see your image in your local Docker daemon.</p>
 
-        <p>Using this technique, CVEs in dependencies with strong compatibility guarantees (e.g., patches to OpenSSL, patches to the JVM) can be mitigated rapidly for many apps.</p>
-        <img src="images/cve.png" alt="Security Patching" class="img-center">
+        <h3 class="pt-5">Running Your Application</h3>
+        <p>Now that you have successfully built your OCI image using Cloud Native Buildpacks you can run your application using the Docker CLI.</p>
 
+        <pre class="command-line code-block"><code>docker run -e PORT=8000 -p 8000:8000 &lt;image name&gt;</code></pre>
 
+        <p>Congratulations! You have now successfully built and run your first application with Cloud Native Buildpacks.</p>
       </section>
-
     </div>
-    <footer class="mt-5">
-      <div class="container pt-5">
-        <p class="text-light">Reach out to the <a href="https://github.com/buildpack" target="blank">Buildpacks org</a> on Github to learn more.</p>
+
+    <footer>
+      <div class="container footer">
+          <a href="https://www.cncf.io/"><img src="/images/cncf-color.svg" class="footer-logo"></a>
+          <p>We are a <a href="https://www.cncf.io/">Cloud Native Computing Foundation</a> hosted project</p>
+      </div>
+      <div class="bg-brand-blue">
+        <div class="container sub-footer">
+          <p>Reach out to <a href="https://github.com/buildpack">Buildpacks.org</a> on GitHub to learn more.</p>
```

Comment:
`Buildpacks.org` should just be `Buildpacks org`, and maybe we should say something more like "open an issue on github"?